pkg/app/app.go

```diff
@@ -131,7 +131,27 @@ func CreateApp() App {
 	return app
 }
 
+// TODO: should this be a go routine?
```

Comment:
Recovery is hard enough. Let's master that before we move to something like concurrent recovery.